infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java

```diff
@@ -88,6 +88,10 @@ public void loadingValidators(final int validatorCount) {
     log.info("Loading {} validator keys...", validatorCount);
   }
 
+  public void atLoadedValidatorNumber(final int loadedValidatorCount) {
+    log.info("Loaded validator number {} ...", loadedValidatorCount);
```

Comment:
Can we make this something like `Loaded validator key 10 of 50`?

Reply:
Makes sense. Done.